geyser-plugin-manager/src/geyser_plugin_service.rs

```diff
@@ -208,4 +150,62 @@ impl GeyserPluginService {
         self.plugin_manager.write().unwrap().unload();
         Ok(())
     }
+
+    fn start_manager_rpc_handler(
+        plugin_manager: Arc<RwLock<GeyserPluginManager>>,
+        request_receiver: Receiver<GeyserPluginManagerRequest>,
+    ) -> Arc<JoinHandle<()>> {
+        Arc::new(thread::spawn(move || loop {
```

Comment:
I believe you need to pass in the exit Atomic variable so that when validator exit it can quit promptly.

Reply:
can you point me to which exit i need to clone? seems like there are a few disconnected Arc<AtomicBool> floating around